source/features/show-names.tsx

```diff
@@ -24,10 +24,15 @@ async function dropExtraCopy(link: HTMLAnchorElement): Promise<void> {
 function appendName(element: HTMLAnchorElement, fullName: string): void {
 	// If it's a regular comment author, add it outside <strong> otherwise it's something like "User added some commits"
 	const {parentElement} = element;
-	const insertionPoint = parentElement!.tagName === 'STRONG' ? parentElement! : element;
+	const isStrong = parentElement?.tagName === 'STRONG';
 
+	if (isStrong) {
+		parentElement?.classList.add('inline');
```

Comment:
`strong` take up more width than its child elements.

Reply:
```suggestion
		parentElement?.classList.add('d-inline-block');
```

GitHub already has a lot of utility classes